core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -858,39 +861,50 @@ public boolean load() {
 
         if (!allowWrites && dataAccessDefaultType.isMMap())
             dataAccessDefaultType = DAType.MMAP_RO;
-        if (encodingManager == null)
-            // we did not call init(), so we build the encoding manager now.
-            // just like when calling init, users have to make sure they use the same setup for import and load
+
```

Comment:
This problem no longer exists. The encoding manager is never created in `init()` so it does not matter whether we called it or not.